src/Resources/skeleton/authenticator/login_form.tpl.php

```diff
@@ -36,6 +59,7 @@
             </label>
         </div>
     #}
```

Comment:
Now that we're asking the question explicitly, if the user chooses "no" to remember me, let's not generate anything (so delete the `else` case and the code above)